plutus-pab/src/Plutus/PAB/Simulator.hs

```diff
@@ -751,7 +752,7 @@ addWalletWith funds = do
         STM.writeTVar _agentStates newWallets
     _ <- handleAgentThread (knownWallet 2) Nothing
             $ Modify.wrapError WalletError
-            $ MockWallet.distributeNewWalletFunds funds (CW.paymentPubKeyHash mockWallet)
+            $ MockWallet.distributeNewWalletFunds def funds (CW.paymentPubKeyHash mockWallet)
```

Comment:
Since you added Params to BlockchainEnv, you can get it from the PABEnvironment.

Reply:
Nice, thanks! I fixed `def` in some places but missed it here. I think we should be more careful with `def`.